src/Symfony/Component/Routing/Annotation/Route.php

```diff
@@ -53,6 +57,26 @@ public function __construct(array $data)
             unset($data['path']);
         }
 
+        if (isset($data['locale'])) {
+            $data['defaults']['_locale'] = $data['locale'];
+            unset($data['locale']);
+        }
+
+        if (isset($data['format'])) {
+            $data['defaults']['_format'] = $data['format'];
+            unset($data['format']);
+        }
+
+        if (isset($data['fragment'])) {
```

Comment:
What's the use case?

Reply:
I don't know if there is one yet, again I propose it for consistency, I can remove it :).

Reply:
We should only expose something if there is a use case. Let's remove everything that has no known use cases.

Reply:
Ok done!